src/clientversion.h

```diff
@@ -17,13 +17,13 @@
  */
 
 //! These need to be macros, as clientversion.cpp's and dynamic*-res.rc's voodoo requires it
-#define CLIENT_VERSION_MAJOR 1
-#define CLIENT_VERSION_MINOR 5
+#define CLIENT_VERSION_MAJOR 2
+#define CLIENT_VERSION_MINOR 0
 #define CLIENT_VERSION_REVISION 0
 #define CLIENT_VERSION_BUILD 0
 
 //! Set to true for release, false for prerelease or test build
-#define CLIENT_VERSION_IS_RELEASE false
```

Comment:
We should wait to merge so that phase 2 test nodes won't build as a release.

Reply:
Yes.